kfs-core/src/main/resources/edu/arizona/kfs/fp/document/datadictionary/DisbursementVoucherDocument.xml

```diff
@@ -102,4 +122,40 @@
           </map>
         </property>
     </bean>
+    
+    <bean id="DisbursementVoucherDocument-dvPaidYear" parent="DisbursementVoucherDocument-dvPaidYear-parentBean"/>
+
+	 <bean id="DisbursementVoucherDocument-dvPaidYear-parentBean" abstract="true" parent="AttributeDefinition">
+	   	<property name="name" value="dvPaidYear"/>
+       	<property name="forceUppercase" value="false"/>	    
+       	<property name="label" value="Calendar Year"/>
+	    <property name="shortLabel" value="Calendar Year"/>
+	    <property name="maxLength" value="4"/>
+	    <property name="required" value="false"/>
+	    <property name="control">
+	    	<bean parent="TextControlDefinition" p:size="5"/>
+	    </property>	    
+    </bean>
+    
+ 	<bean id="DisbursementVoucherDocument-dv1099Ind" parent="DisbursementVoucherDocument-dv1099Ind-parentBean"/>
+
+  	<bean id="DisbursementVoucherDocument-dv1099Ind-parentBean" abstract="true" parent="GenericAttributes-genericBoolean">
+    	<property name="name" value="dv1099Ind"/>
+ 	    <property name="label" value="1099 Classification"/>
+    	<property name="shortLabel" value="1099 Classification"/>
+    	<property name="control">
+      		<bean parent="RadioControlDefinition" p:valuesFinderClass="org.kuali.rice.krad.keyvalues.IndicatorValuesFinder" p:includeKeyInLabel="false"/>
+    	</property>
+  	</bean>
+  	    
+    <bean id="DisbursementVoucherDocument-dvSpecialHandlingCodeForSearching" parent="DisbursementVoucherDocument-dvSpecialHandlingCodeForSearching-parentBean"/>
```

Comment:
This should be DisbursementVoucherDocument-dv1099IndForSearching. DisbursementVoucherDocument-dvSpecialHandlingCodeForSearching is from another mod.